processor/src/decorators/mod.rs

```diff
@@ -53,7 +53,7 @@ where
     // --------------------------------------------------------------------------------------------
 
     /// Injects a node of the Merkle tree specified by the values on the stack at the head of the
-    /// advice tape. The stack is expected to be arranged as follows (from the top):
+    /// advice stack. The stack is expected to be arranged as follows (from the top):
```

Comment:
Let's update: "at the head of the advice stack" -> "at the top of the advice stack".